src/language-handlebars/printer-glimmer.js

```diff
@@ -622,5 +609,5 @@ function isPrettierIgnoreNode(node) {
 
 module.exports = {
   print,
-  massageAstNode: clean
+  massageAstNode: require("./clean")
```

Comment:
move to top?

Reply:
I'm not sure

Reply:
I mean `const clean = require("./clean")` at top of this file